.npmrc

```diff
@@ -1,6 +1,6 @@
 disturl="https://electronjs.org/headers"
-target="34.5.1"
-ms_build_id="11369351"
+target="35.1.5"
```

Comment:
35.2.0?

Reply:
Isn't it available already?
https://releases.electronjs.org/release/v35.2.0
https://www.npmjs.com/package/electron
(I do not use npm/node usually so I may not know how things work exactly)